### PR TITLE
fix: issue-609

### LINKED ISSUE #609: The description of first post in Revision select is unclear
**To Reproduce**

Edit question/answer/tag:
1. Select 'revision'
2. The first revision's description is "Edit xxx" 

**Expected behavior**

The first revision's description is "Add xxx"
- Add question
- Add answer
- Add tag

**Screenshots**

image

### PR DESCRIPTION
**Pull request by kelvinkuo**
fix #609

### CODE REVIEW

File: i18n/nl_NL.yaml [hunk 1]
```diff
@@ -463,7 +463,8 @@ ui:
   edit_tag:
     title: Edit Tag
     default_reason: Edit tag
-    form:
+    default_first_reason: Add tag
+    default_first_reason: Add tag    form:
```
Comment (shuashuai) [inline i18n/nl_NL.yaml:467]: It’s repeated here. You need to modify it. Also pay attention to the format.

**Review by shuashuai — CHANGES_REQUESTED** [review]
First of all, thank you very much for your contribution. What you need to know about i18n files and how to contribute.

Our translation uses [crowdin](https://crowdin.com/project/answer), so if you contribute by changing the language file in the code, then you only need to change en_US.yaml, other files will be overwritten after they are synchronized on crowdin.

**Comment by kelvinkuo** [comment]
@shuashuai Thanks for your expaination, please review one more time.

**Review by shuashuai — APPROVED** [review]
That's great,we will merge this PR as soon as possible.

**Pull request merged**